### ISSUE TITLE
symfony 4, missing commands from console

### ISSUE THREAD

**Issue by FirstSailor** [Phobetor/rabbitmq-supervisor-bundle#31]
None of the RabbitMQ supervisor bundle commands work on symfony4

**Comment by stanislas-prime** [comment]
Need that, too. Thanks

**Comment by FirstSailor** [comment]
managed to load commands by adding them to services.yaml:
```
    Phobetor\RabbitMqSupervisorBundle\Command\:
        resource: '../vendor/phobetor/rabbitmq-supervisor-bundle/Phobetor/RabbitMqSupervisorBundle/Command/*'
```

but then another problem occurs:
```
The "phobetor_rabbitmq_supervisor" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead. 
```

**Comment by Phobetor** [comment]
Hi @FirstSailor  

Thank you very much for this pull request!

It is merged to dev-master and released in version [3.0.0](https://github.com/Phobetor/rabbitmq-supervisor-bundle/releases/tag/3.0.0).
